mautic/mautic
# Updating the email validation mode to work in Symfony 5.0

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-301



#### Description:

The strict mode requires another dependency to be added via Composer so I suggest to change it to the next best thing which is the same email validation as the HTML5 browsers do.

See https://symfony.com/doc/current/reference/constraints/Email.html#mode

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test email validation in the global configuration
3. Go to the Email Settings
4. Test that you cannot save any rubbish value to the **E-mail address to send mail from** and **Reply to address** fields. It will allow only email an address.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11054"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
